docs/misc/faq.md

```diff
@@ -0,0 +1,110 @@
+---
+title: FAQ
+---
+
+Other questions? Ask on `#wg-developer-stonesoup`.
```

Comment:
Do you want to link to the channel? Call out that this is on the internal Slack instance? Is there a more public place/method for asking external to Red Hat?

Reply:
this is a good question, I don't think that the `infra-deployment` repo is relevant to people outside of Red Hat, so we don't need to link an external slack channel.